estela-web/src/components/SpiderDetailPage/index.tsx

```diff
@@ -95,29 +156,53 @@ export class SpiderDetailPage extends Component<RouteComponentProps<RouteParams>
             this.apiService.apiProjectsSpidersRead(requestParams).then(
                 async (response: Spider) => {
                     const data = await this.getSpiderJobs(1);
+                    const completedJobs = data.data.filter((job: SpiderJobData) => job.jobStatus === "COMPLETED");
+                    const runningJobs = data.data.filter((job: SpiderJobData) => job.jobStatus === "RUNNING");
+                    const queueJobs = data.data.filter((job: SpiderJobData) => job.jobStatus === "IN_QUEUE");
+                    const tableStatus = [
+                        !(queueJobs.length === 0),
+                        !(runningJobs.length === 0),
+                        !(completedJobs.length === 0),
+                    ];
                     const jobs: SpiderJobData[] = data.data;
                     this.setState({
                         name: response.name,
                         jobs: [...jobs],
                         count: data.count,
                         current: data.current,
                         loaded: true,
+                        tableStatus: [...tableStatus],
+                        queueJobs: [...queueJobs],
+                        runningJobs: [...runningJobs],
+                        completedJobs: [...completedJobs],
                     });
                 },
                 (error: unknown) => {
                     console.error(error);
                     resourceNotAllowedNotification();
                 },
             );
+            const requestParamsDeploys: ApiProjectsDeploysListRequest = { pid: this.projectId };
+            this.apiService.apiProjectsDeploysList(requestParamsDeploys).then(
+                (results) => {
+                    const deploys: Deploy[] = results.results;
+                    const lastDeployDate: Date =
+                        deploys.reduce((d1, d2) => {
+                            const date1: Date = d1?.created || new Date();
+                            const date2: Date = d2?.created || new Date();
+                            return date1 > date2 ? d1 : d2;
+                        })?.created || new Date();
+
+                    this.setState({ lastDeployDate: convertDateToString(lastDeployDate) });
+                },
+                (error: unknown) => {
+                    console.error(error);
+                    resourceNotAllowedNotification();
+                },
+            );
```

Comment:
This value math the creation of the spider? I think you can get this value other easy way only with the spider ID

Reply:
Changed spider creation date logic established on the meeting.